app/views/public/api/v1/models/_category.json.jbuilder

```diff
@@ -1,5 +1,25 @@
-json.id category.id
 json.name category.name
+json.slug category.slug
 json.locale category.locale
 json.description category.description
 json.position category.position
+
+json.related_categories do
+  if category.related_categories.any?
+    json.array! category.related_categories.each do |related_category|
+      json.partial! partial: 'associated_category', category: related_category
+    end
+  end
+end
+
+if category.parent_category.present?
```

Comment:
root ?

Reply:
No no, we have 3 different category associations
1. Parent - sub categories
2. linked_categories which we are going to change to associated categories.
3. related categories with many-to-many associations mostly work like tags.